cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/MarthaHttpRequestRetryStrategy.scala

```diff
@@ -0,0 +1,51 @@
+package cloud.nio.impl.drs
+
+import java.io.IOException
+
+import org.apache.http.HttpResponse
+import org.apache.http.client.{HttpRequestRetryHandler, ServiceUnavailableRetryStrategy}
+import org.apache.http.protocol.HttpContext
+
+import scala.concurrent.duration._
+
+class MarthaHttpRequestRetryStrategy(drsConfig: DrsConfig)
+  extends ServiceUnavailableRetryStrategy with HttpRequestRetryHandler {
+
+  // We can execute a total of one time, plus the number of retries
+  private val executionMax: Int = drsConfig.numRetries + 1
+  private val waitMultiplier: Double = drsConfig.waitMultiplier
+  private val waitMax: FiniteDuration = drsConfig.waitMaximum
+  private var waitNext: Duration = drsConfig.waitInitial min waitMax
```

Comment:
TOL perhaps `SimpleExponentialBackoff` could help here?

Reply:
Good catch. Better to not add yet another implementation.

However, this artifact does not currently depend on `core`, and all of its dependencies. Someday I'd like to slim down the "simple" localizer. For example: As of today the [localizer image](https://hub.docker.com/repository/docker/broadinstitute/cromwell-drs-localizer/tags?page=1&name=develop) is 1.5 times the size of the [server image](https://hub.docker.com/repository/docker/broadinstitute/cromwell/tags?page=1&name=develop), and one could argue does a LOT less. So I don't want to explicitly pull in `core` unless one argues it's necessary. If anything would like to slim down the image so we don't eventually blow out the default 10GB disk for small Papi jobs by transitively including the entirety of mvnrepository.com.

Good news: In the Cromulent spirit of using both `akka-streams` & `fs2`, `circe` & `spray-json`, `cats` & `scalaz`, etc.-- there is _of course_ (at least?) two implementations of exponential backoffs in the codebase. I used a different one than the one you recommended, updating it to accept not-randomizing so I could simplify the existing tests. Let me know your thoughts.